wiki/en/en-Walk-Through.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: wiki
+title: "Jamulus Walk-Through"
+lang: "en"
+permalink: "/wiki/Walk-Through"
+---
+# Jamulus walk-through
+Let's have a look at what Jamulus looks like and what you should do after having installed the software.
+## First start
+Before you start Jamulus:
+1. Close all other applications
+2. Plugin everything (Ethernet cable, audio gear, etc.)
```

Comment:
Either "Plug in everything" or (my preference to avoid slight possibility of misunderstanding with the word "plugin" which means something very different) "Connect everything"

Reply:
Yes. Connect everything is better.

Reply:
Changed